Makefile.dep

```diff
@@ -1065,6 +1065,7 @@ ifneq (,$(filter xtimer,$(USEMODULE)))
       FEATURES_REQUIRED += periph_timer
     else
       # will use *ztimer_usec as low-level timer*
+      USEMODULE += ztimer_usec
```

Comment:
Does this make sense, `ztimer`'s `Makefile.dep` already has this

```
# make xtimer use ztimer_usec as low level timer
ifneq (,$(filter xtimer_on_ztimer,$(USEMODULE)))
  USEMODULE += ztimer_usec
endif
```

Reply:
Strange! I added this because I ended up with undefined references. I assume with only `xtimer_on_ztimer` and no other ztimer related module used, `ztimer`'s `Makefile.dep` does not get parsed. I will fix tonight (or feel free to just (force) push into this branch, if you want)